### PR TITLE
Add passed problem bar & URL parser

### PR DESCRIPTION
**Pull request by tiger2005**

### CODE REVIEW

File: src/components/InfoCard.tsx [hunk 1]
```diff
@@ -270,10 +270,33 @@ const PassedProblemsInfo = (props: {info: number[]}) => {
   );
 };
 
+const UidDisplayer = (props: {num: number}) => {
+  // definition of ".uidDisplayer" => cardStyle.ts
+  // I want CSS3 !!! /ll
```
Comment (Nikaidou-Shinku) [inline src/components/InfoCard.tsx:292]: Well, as what I had said I will put Fluogu on hold for a while.

Comment (tiger2005) [reply src/components/InfoCard.tsx:292]: I knew it, just for exercise. Now I can quickly make this stuff with CSS.
Actually, the function is from a request by @songhongyi.

**Comment by tiger2005** [comment]
最后 merge 完成，发布一个 Release，就去 LibGu 干活了（只要有时间的话）

**Comment by Nikaidou-Shinku** [comment]
项目重构了，在新版本上重写吧.jpg

**Pull request closed without merging**